interfaces/builtin/polkit.go

```diff
@@ -150,9 +150,9 @@ func (iface *polkitInterface) BeforePreparePlug(plug *snap.PlugInfo) error {
 }
 
 func isPathMountedWritable(mntProfile *osutil.MountProfile, fsPath string) bool {
-	mntMap := make(map[string]*osutil.MountEntry, len(mntProfile.Entries))
+	mntMap := make(map[string]osutil.MountEntry, len(mntProfile.Entries))
 	for _, mnt := range mntProfile.Entries {
-		mntMap[mnt.Dir] = &mnt
+		mntMap[mnt.Dir] = mnt
 	}
```

Comment:
maybe this would work instead:

```
	for i := range mntProfile.Entries {
                mnt := &mntProfile.Entries[i]
		mntMap[mnt.Dir] = mnt
	}
```

Reply:
good point, this is less wasteful, thank you